models/flower.py

```diff
@@ -0,0 +1,13 @@
+from odoo import fields, models
+
+
+class Flower(models.Model):
+    _name = 'flower'
+    _description = 'Flower'
+
+    name = fields.Char(required=True)
+    scientific_name = fields.Char()
+    season_date_start = fields.Date()
+    season_date_end = fields.Date()
+    watering_frequency = fields.Integer(help='Should be watered once every x days')
+    watering_amount = fields.Float(help='In milliliters')
```

Comment:
The definition of this float field is perfectly fine, aside from the lack of a label. An advice, though, usually it is necessary to limit how many digits can be represented through a float field. This can be done through the **digits** attribute.

Example: 
```time_spent = fields.Float('Time', digits=(16, 2))```

Also, if you wish to use the milliliters unit of measure (UoM), create a new Many2One field that references the Unit of Measure:

```watering_amount_uom = fields.Many2one('uom.uom', 'Unit of Measure', readonly=True)```

The Unit of Measure set on this field will be used to format the amount on the Form view.